client/__tests__/e2e/e2eAnnotations.test.js

```diff
@@ -183,8 +183,7 @@ describe.each([
   });
 
   async function assertCategoryExists(categoryName) {
-    const result = await utils.waitByID(`${categoryName}:category-expand`);
-    expect(await result.evaluate(node => node.innerText)).toBe(categoryName);
+    await utils.waitByID(`${categoryName}:category-expand`);
```

Comment:
rather than deleting the `expect`, can we do a weaker test?  For example, test that the first 10 and last 10 characters match, or something like that?

Reply:
What I had in mind (sorry if it contains typos):

```
const foundCatName = await result.evaluate(node => node.innerText);
expect(foundCatName.slice(0, 10)).toBe(categoryName.slice(0, 10))
expect(foundCatname.slice(-10).toBe(categoryName.slice(-10))
```

Reply:
or pick a number (instead of 10) that matches closer to what you expect.

Reply:
@colinmegill sounds like our second solution might be worthwhile

Reply:
@mweiden - if I'm mistaken, and this _is_ being tested elsewhere, feel free to land the PR.  It just seemed easy to make the test almost as robust as it was previously.  LMK if I'm off base!

Reply:
@bkmartinjr slight misunderstanding here: `waitByID` throws an exception if it doesn't find the element it is looking for, so the test would fail.